docs/api/doc-block-markdown.md

```diff
@@ -26,8 +26,20 @@ import { Button } from "@storybook/design-system";
 ```md
```

Comment:
@rwaight we have a bit of room for improvement here. We can adjust the snippet to the following:

````md
<!-- prettier-ignore-start -->
```md
<!-- Header.mdx -->

// DON'T do this, will error
import ReadMe from './README.md';
// DO this, will work
import ReadMe from './README.md?raw';

import { Markdown } from '@storybook/blocks';

# A header 

<Markdown>{ReadMe}</Markdown>
```
<!-- prettier-ignore-end -->
````

With this small change, the snippet will be rendered as follows and we avoid having broken down into two while retaining the required information.

![CleanShot 2024-02-02 at 15 56 43@2x](https://github.com/storybookjs/storybook/assets/22988955/eb42eb48-3170-4a08-bf72-c0b6eb463c39)